Snippets/Snippets_4/Host/EndpointStartAndStop.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Snippets4.Host
+{
+    using NServiceBus;
+
+    #region host-EndpointStartAndStop
```

Comment:
We split the snippets, one to show generic lifecycle presented by the core (lifecycle-EndpointStartStop) and lifecycle usable in the host (host-EndpointStartAndStop)